### ISSUE TITLE
drizzle cannot run on node when using server side rendering

### ISSUE THREAD

**Issue by TovarishFin** [ConsenSys-archive/drizzle-legacy#104]
I ran into an issue with the latest update which I believe has to do with last releases's webpack update. This can be easily resolved by simply updating webpack config with the following:
```
output: {
...
  globalObject: 'typeof self !== \'undefined\' ? self : this',
...
},
```
I believe this is is an issue relating to https://github.com/webpack/webpack/issues/6784

When updating the config and building on my machine, it seems to work fine.

**Comment by vmaykovsky** [comment]
I've got the same issue working with Next.js + Drizzle. @TovarishFin, your approach helps but doesn't fix the issue completely. I think this is quite a serious issue, which should be fixed ASAP.

**Comment by adrianmcli** [comment]
@TovarishFin would love a PR for this if you already have a working solution.

**Comment by soundyogi** [comment]
this seems mergable to me but the /dist version is also included in the commit.

**Comment by adrianmcli** [comment]
There's no PR attached to this. If @yvon would like to make a PR for this, I'd be happy to review it.

**Comment by yvon** [comment]
@adrianmcli give me few days. I have to focus on it.

This quick fix did the job for us, but I'm not sure it's the most elegant approach. I have also few more fixes dealing with the window object in my branch, done by @vedi. It's worth considering creating PRs for those too.

**Comment by chrischip** [comment]
#153 Please review this PR I believe this is the fix

**Comment by cds-amal** [comment]
Closing this as it is resolved by #153.
